packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -43,8 +61,10 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
     const corePackageJsonContent = JSON.parse(fs.readFileSync(path.resolve(__dirname, '..', '..', 'package.json'), {encoding: 'utf-8'})) as PackageJson;
     const o3rCoreVersion = corePackageJsonContent.version ? `@${corePackageJsonContent.version}` : '';
     const schematicsDependencies = ['@o3r/dev-tools', '@o3r/schematics'];
+    const packageManager = getPackageManagerFromTree(tree);
     for (const dependency of schematicsDependencies) {
       context.addTask(new DevInstall({
+        packageManager,
```

Comment:
You should use the one exposed by `@o3r/schematics` that determine the packageManager automatically

Reply:
At this moment 'o3r/schematics' is not installed. We call the `DevInstall` constructor to install schematics and dev-tools. 
I find it a bit strange because `o3r/schematics` is a dep for `o3r/core` and 'o3r/dev-tools' is a dep for 'o3r/schematics'. I think it is for 'pnp' that needs to have them explicitly in `package.json`.
@cpaulve-1A Not sure if this sounds familiar to you :)

Reply:
`o3r/schematics` is in prod dep of `o3r/core`, it should be installed :S

Reply:
Indeed. I gave it a try.

Reply:
finally done in https://github.com/AmadeusITGroup/otter/pull/473